neqo-transport/src/connection.rs

```diff
@@ -2568,6 +2582,46 @@ mod tests {
         assert!(server.crypto.tls.info().unwrap().resumed());
     }
 
+    fn connect_with_rtt(client: &mut Connection, server: &mut Connection, now: &mut Instant) {
+        const TIME_INTERVAL: u64 = 20;
+
+        let mut a = client;
+        let mut b = server;
+        let mut datagram = None;
+        let is_done = |c: &mut Connection| match c.state() {
+            State::Confirmed | State::Closing { .. } | State::Closed(..) => true,
+            _ => false,
+        };
+
+        while !is_done(a) {
+            let _ = maybe_authenticate(a);
+            let d = a.process(datagram, *now);
+            datagram = d.dgram();
+            *now += Duration::from_millis(TIME_INTERVAL);
+            mem::swap(&mut a, &mut b);
+        }
+        a.process(datagram, *now);
```

Comment:
It is probably worth testing that the RTT is accurate here.

```suggestion
        a.process(datagram, *now);

        assert_eq!(a.loss_recovery.rtt(), RTT);
        assert_eq!(b.loss_recovery.rtt(), RTT);
```